src/js/components/user-management/selfusermanagement.js

```diff
@@ -189,6 +189,7 @@ const mapStateToProps = state => {
   return {
     has2FA: state.users.globalSettings.hasOwnProperty('2fa') && state.users.globalSettings['2fa'] === 'enabled',
     isEnterprise: state.app.features.isEnterprise || (state.app.features.isHosted && plan !== 'os'),
+    isHosted: state.app.features.isEnterprise || state.app.features.isHosted,
```

Comment:
isHosted as it is now is misleading, since `isEnterprise` can be true for on-prem installations. and we handle those as normal "hosted" with `isHosted === ture`?

Reply:
this one and the next comment are very valid points, however the structure of this "feature flag" is a bit special, since it should show a note in commercial, but not in enterprise & professional (so here these two are considered "enterprisey")... but you're right, I'll reword this...